tfs/src/main/java/hudson/plugins/tfs/util/BuildWorkspaceConfigurationRetriever.java

```diff
@@ -45,17 +45,23 @@ public BuildWorkspaceConfiguration getLatestForNode(Node needleNode, Run<?,?> la
     
     public static class BuildWorkspaceConfiguration extends WorkspaceConfiguration {
         private static final long serialVersionUID = 1L;
-        private final AbstractBuild<?, ?> build;
+        private final transient AbstractBuild<?, ?> build;

```

Comment:
Actually due to lazy loading issues there are cases where including a non`transient` field of type `Run` will cause serious errors, so it is mandatory to make this `transient`.